src/System.Private.CoreLib/src/System/Runtime/InteropServices/Marshal.cs

```diff
@@ -559,9 +505,11 @@ private static unsafe T ReadValueSlow<T>(object ptr, int ofs, Func<IntPtr, int,
         {
             // Consumers of this method are documented to throw AccessViolationException on any AV
             if (ptr == null)
+            {
                 throw new AccessViolationException();
+            }
 
-            int dwFlags = 
+            const int Flags = 
```

Comment:
`flags`

Reply:
I fixed this according to the code guidelines (https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/coding-style.md)

> We use PascalCasing to name all our constant local variables and fields.

Reply:
Oh... since it is constant. Alright.